test/ivToken/IvTokenMint.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.16;
+
+import "forge-std/Test.sol";
+import "../TestCommons.sol";
+import "../../contracts/tokens/IvToken.sol";
+
+contract IvTokenMintTest is Test, TestCommons {
+    IvToken internal _ivToken;
+    address internal _admin;
+    address internal _userOne;
+    address internal _userTwo;
+
+    event Mint(address indexed account, uint256 amount);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+
+    function setUp() public {
+        _ivToken = new IvToken("IvToken", "IVT", address(0x6B175474E89094C44Da98b954EedeAC495271d0F));
+        _admin = address(this);
+        _userOne = _getUserAddress(1);
+        _userTwo = _getUserAddress(2);
+    }
+
+	function testShouldNotMintIvTokenWhenNotStanley () public {
+		// given
+		// when
+		// then
+		vm.expectRevert(abi.encodePacked("IPOR_501"));
+		_ivToken.mint(_userOne, 1000000000000000000000000);
+	}
+
+	function testShouldNotMintIvTokenWhenAmountIsZero() public {
+		// given
+		address mockIporVaultAddress = _userOne;
+		_ivToken.setStanley(mockIporVaultAddress);
+		// when
+		vm.prank(_userOne);
```

Comment:
this is only setup sender,

Reply:
I set up the sender to get `IPOR_503`. Otherwise I get `IPOR_501`

Reply:
I know, but this is not the main action in the test. In the `when` section, there should be `mint` token

Reply:
Do you mean this?

```
	function testShouldNotMintIvTokenWhenAmountIsZero() public {
		// given
		address mockIporVaultAddress = _admin;
		_ivToken.setStanley(mockIporVaultAddress);
		// when
		_ivToken.mint(_userOne, 1*10**18);
		vm.expectRevert(abi.encodePacked("IPOR_503"));
		_ivToken.mint(_userOne, 0);
	}
```

Reply:
i would say 
```
	function testShouldNotMintIvTokenWhenAmountIsZero() public {
		// given
		address mockIporVaultAddress = _admin;
		_ivToken.setStanley(mockIporVaultAddress);
		// when
		vm.expectRevert(abi.encodePacked("IPOR_503"));
		_ivToken.mint(_userOne, 0);
	}
```